packages/ubuntu_provision/lib/src/services/config_service.dart

```diff
@@ -123,7 +124,9 @@ class ConfigService {
   @visibleForTesting
   static String? lookupPath(FileSystem fs) {
     for (final ext in _extensions) {
-      final path = join(whiteLabelDirectory, '$_filename.$ext');
+      final path = join(
+          Platform.environment['DESKTOP_PROVISION_PATH'] ?? whiteLabelDirectory,
+          '$_filename.$ext');
```

Comment:
```suggestion
          '$_filename.$ext',);
```
Add trailing comma, needs formatting afterwards